### PR TITLE
Shorten line length of method declaration

### PR DESCRIPTION
**Pull request by anoadragon453**
~~Adds parameters to `CheckKeys` that lets you specify whether well-known is being used or not, and the new server name if so.~~

~~https://github.com/matrix-org/matrix-federation-tester/pull/19~~

Simply does some aesthetics updates now as all PR functionality was moved to matrix-federation-tester.

### CODE REVIEW

File: keys.go [hunk 1]
```diff
@@ -173,10 +173,18 @@ type KeyChecks struct {
...
+	keys ServerKeys,
+	connState *tls.ConnectionState,
+	wellKnown *WellKnownResult,
+) (
+	checks KeyChecks,
+	ed25519Keys map[KeyID]Base64String,
+	sha256Fingerprints []Base64String,
 ) {
-	checks.MatchingServerName = serverName == keys.ServerName
+	checks.MatchingServerName = serverName == keys.ServerName || serverName == wellKnown.NewAddress
```
Comment (richvdh) [inline keys.go]: in any case: the returned server name should *never* be the NewAddress returned by the well-known.

**Comment by anoadragon453** [comment]
Un-done weirdness with this check, thanks for the explanation.

I've now broken up serverName and serverHost into two separate vars, though currently none of the checks really care (afaict) if the serverName is different from the matrix server's host. So I've left the breakup there in case we need a distinction in the future. (If you think this is weird I can just remove it).

File: keys.go [hunk 2]
```diff
@@ -161,9 +161,14 @@ type KeyChecks struct {
 }
 
 // CheckKeys checks the keys returned from a server to make sure they are valid.
-// If the checks pass then also return a map of key_id to Ed25519 public key
-func CheckKeys(serverName ServerName, now time.Time, keys ServerKeys) (
-	checks KeyChecks, ed25519Keys map[KeyID]Base64String,
+// If the checks pass then also return a map of key_id to Ed25519 public key and a list of SHA256 TLS fingerprints.
```
Comment (richvdh) [inline keys.go]: It doesn't return the TLS fingerprints afaict?

**Comment by anoadragon453** [comment]
Erm, yep, that's all that's left of this PR. LGTY?

**Comment by richvdh** [comment]
well, I guess. It seems a bit pointless tbh, but if you want to keep it, change the PR title.

**Pull request merged**